db/schema.rb

```diff
@@ -97,6 +97,13 @@
     t.index ["title"], name: "index_leaderships_on_title", unique: true
   end
 
+  create_table "pay_scales", id: :uuid, default: -> { "gen_random_uuid()" }, force: :cascade do |t|
+    t.string "label", null: false
```

Comment:
For now, just remove `null: false` and we can use a validation.  It'll stop it failing.

Reply:
To expedite, though, don't bother with the validation unless it's already there.  Put in a ticket to add it later.